### ISSUE TITLE
Readme and License. 

### ISSUE THREAD

**Issue by nawazdhandala** [CloudBoost/dashboard-ui#11]

**Comment by crazy4groovy** [comment]
Yes please! I got to this repo by searching around after seeing this response:

`{"status":200,"version":"1.0.23","message":"This is CloudBoost API. If you're looking for the dashboard. It should be running on port 1440."}`

`npm install && node server.js` got a server running on post 1440, but the page looks blanks. Am I missing a build step?

`Cannot GET /.build/js/app.bundle.js`

**Comment by crazy4groovy** [comment]
(Linting error found in dashboard-ui/app/actions/index.js line:880)

**Comment by nawazdhandala** [comment]
@crazy4groovy  This repo is under active development. If you're looking for dashboard please check cloudboost/dashboard out. 

Let me know if this helps.

**Comment by ritishgumber** [comment]
done.
